Marlin/Marlin_main.cpp

```diff
@@ -9436,6 +9436,39 @@ inline void gcode_M503() {
 
 #endif // ADVANCED_PAUSE_FEATURE
 
+#if ENABLED(MK2_MULTIPLEXER)
+
+  inline void select_multiplexed_stepper(const uint8_t e) {
+    stepper.synchronize();
+    disable_e_steppers();
+    WRITE(E_MUX0_PIN, TEST(e, 0) ? HIGH : LOW);
+    WRITE(E_MUX1_PIN, TEST(e, 1) ? HIGH : LOW);
+    WRITE(E_MUX2_PIN, TEST(e, 2) ? HIGH : LOW);
```

Comment:
Should probably only write to `E_MUX1_PIN` and `E_MUX2_PIN` if they are defined.